[2023-07-11 10:49:07,566] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_user 2023-06-17T08:01:18+00:00 [queued]>
[2023-07-11 10:49:07,665] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_user 2023-06-17T08:01:18+00:00 [queued]>
[2023-07-11 10:49:07,666] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-07-11 10:49:07,666] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2023-07-11 10:49:07,667] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-07-11 10:49:07,729] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): create_user> on 2023-06-17T08:01:18+00:00
[2023-07-11 10:49:07,737] {standard_task_runner.py:52} INFO - Started process 12621 to run task
[2023-07-11 10:49:07,751] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'create_user', '2023-06-17T08:01:18+00:00', '--job-id', '6319', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag/test.py', '--cfg-path', '/tmp/tmp77l35dvk', '--error-file', '/tmp/tmpmoljga8a']
[2023-07-11 10:49:07,753] {standard_task_runner.py:77} INFO - Job 6319: Subtask create_user
[2023-07-11 10:49:07,867] {logging_mixin.py:104} INFO - Running <TaskInstance: my_first_dag.create_user 2023-06-17T08:01:18+00:00 [running]> on host d9c3ec86c3ab
[2023-07-11 10:49:08,217] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=create_user
AIRFLOW_CTX_EXECUTION_DATE=2023-06-17T08:01:18+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-17T08:01:18+00:00
[2023-07-11 10:49:08,220] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-07-11 10:49:08,221] {subprocess.py:63} INFO - Running command: ['bash', '-c', '\nid -u test_user;\nif [$? -eq o];\n    then echo "User exists"\nelse sudo useradd -r test_user;\n    echo "User created";\nfi']
[2023-07-11 10:49:08,240] {subprocess.py:75} INFO - Output:
[2023-07-11 10:49:08,328] {subprocess.py:79} INFO - id: ‘test_user’: no such user
[2023-07-11 10:49:08,336] {subprocess.py:79} INFO - bash: line 2: [1: command not found
[2023-07-11 10:49:08,412] {subprocess.py:79} INFO - 
[2023-07-11 10:49:08,413] {subprocess.py:79} INFO - We trust you have received the usual lecture from the local System
[2023-07-11 10:49:08,414] {subprocess.py:79} INFO - Administrator. It usually boils down to these three things:
[2023-07-11 10:49:08,414] {subprocess.py:79} INFO - 
[2023-07-11 10:49:08,415] {subprocess.py:79} INFO -     #1) Respect the privacy of others.
[2023-07-11 10:49:08,416] {subprocess.py:79} INFO -     #2) Think before you type.
[2023-07-11 10:49:08,416] {subprocess.py:79} INFO -     #3) With great power comes great responsibility.
[2023-07-11 10:49:08,417] {subprocess.py:79} INFO - 
[2023-07-11 10:49:08,418] {subprocess.py:79} INFO - sudo: no tty present and no askpass program specified
[2023-07-11 10:49:08,428] {subprocess.py:79} INFO - User created
[2023-07-11 10:49:08,431] {subprocess.py:83} INFO - Command exited with return code 0
[2023-07-11 10:49:08,498] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=create_user, execution_date=20230617T080118, start_date=20230711T104907, end_date=20230711T104908
[2023-07-11 10:49:08,580] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-11 10:49:08,610] {local_task_job.py:151} INFO - Task exited with return code 0
